bin/cli.js

```diff
@@ -70,6 +75,11 @@ async function initKnex(env, opts) {
     config.migrations.directory = null;
   }
 
+  // Client gets defaulted if undefined and it's allowed
+  if(useDefaultClientIfNotSpecified && config.client === undefined){
```

Comment:
Spaces missing around parens.

Reply:
Updated to fix this here and one other place.